src/components/layouts/AddressOverview.vue

```diff
@@ -219,6 +217,21 @@ export default defineComponent({
         const unclaimedCashlinkCount = ref(0);
         const showUnclaimedCashlinkList = ref(false);
 
+        onMounted(() => {
+            // Using setTimeout just to make sure, this is executed at the end of stack call as sometimes, it adds the
+            // 'masked' class although it's not needed
+            setTimeout(() => maskAddress(), 1);
+
+            window.addEventListener('resize', maskAddress);
+        });
+        onUnmounted(() => window.removeEventListener('resize', maskAddress));
+
+        function maskAddress() {
+            const addressElement = context.refs.$address as HTMLDivElement;
+            const masked = addressElement.clientWidth < addressMaskedWidth.value;
+            addressElement.classList[masked ? 'add' : 'remove']('masked');
```

Comment:
Please don't use this programmatical way of setting classes, when we use Vue. Vue is reactive, you just set a `boolean` variable and specify the class in the template. When doing it the way you have here, it becomes very hard to see where classes come from.

See also the commit that introduced the `vue-responsive-components` in the first place: https://github.com/nimiq/wallet/commit/e3e6f3fd18ba9cb8baae0477f3c8376bd743670f

Plus, there would be a method for this:
```diff
- addressElement.classList[masked ? 'add' : 'remove']('masked');
+ addressElement.classList.toggle('masked', masked);
```

Reply:
You are completely right. I am going to change the PR now. Huge mistake 😑